node/bft/src/gateway.rs

```diff
@@ -300,6 +305,12 @@ impl<N: Network> Gateway<N> {
         &self.resolver
     }
 
+    #[cfg(feature = "telemetry")]
+    /// Returns the validator telemetry.
```

Comment:
```suggestion
    /// Returns the validator telemetry.
    #[cfg(feature = "telemetry")]
```
ditto

Reply:
(applies to all the remaining cases)

Reply:
Resolved with [2ba93d8](https://github.com/ProvableHQ/snarkOS/pull/3516/commits/2ba93d833ab3a0017bca6a071d8260838eac4521)